kolibri/plugins/coach/assets/src/routes/baseRoutes.js

```diff
@@ -1,21 +1,23 @@
 import { PageNames } from '../constants';
+import { LessonsPageNames } from '../constants/lessonsConstants';
+import routes from '.';
```

Comment:
The purpose of the baseRoutes is to avoid importing the full routes module into the CoachSideNavEntry module (which is an entry point in itself), as doing so will significantly increase the size of that bundle. So we should undo this import and update the paths in another way.

Reply:
Oooh, didnt know that, I just wanted to avoid the duplication of these routes. But if its because of performance reasons, I think its better just to revert it and keep the old strategy. At the end we wont be touching this file too much.

Reply:
If you want to have a single source of truth, you could put the path segments into a constants file and reference those in here (although that would be further obfuscating the actual paths in multiple places!)

Reply:
I think having a single source of truth might be useful, but I have struggled quite a bit with some of the use of constants throughout the coach pages/routing. I know this refactor overall will help a lot with the parts of this plugin that are more confusing. But, I think if we do go the constants route (...that joke was just for you Richard), it might be helpful to have someone less familiar with the coach architecture in general take a look before approval/merge and see how easy it is for them to parse.

Reply:
> But, I think if we do go the constants route (...that joke was just for you Richard)

This is a very thinly sliced double meaning.